src/control/lib/control/network.go

```diff
@@ -224,9 +225,11 @@ type (
 	}
 
 	GetAttachInfoResp struct {
-		ServiceRanks  []*PrimaryServiceRank `json:"rank_uris"`
-		MSRanks       []uint32              `json:"ms_ranks"`
-		ClientNetHint ClientNetworkHint     `json:"client_net_hint"`
+		ServiceRanks            []*PrimaryServiceRank `json:"rank_uris"`
+		AlternateServiceRanks   []*PrimaryServiceRank `json:"secondary_rank_uris"`
```

Comment:
So AlternativeServiceRanks are a second list of PrimaryServiceRanks rather than a list of SecondaryServiceRanks because they are still PSRs right?

Reply:
Exactly. Calling them "SecondaryServiceRanks" seemed really confusing.